index.html

```diff
@@ -14,41 +15,139 @@
     />
 
   -->
-    <link rel="stylesheet" href="style.css" />
-
-    <script
-      src="https://kit.fontawesome.com/26c38390b4.js"
-      crossorigin="anonymous"
-    ></script>
-  </head>
-  <body>
-    <header class="navbar">
-      <a href="#" class="brand-title">My logo</a>
-      <i class="fas fa-bars"></i>
-    </header>
+  <link rel="stylesheet" href="style.css" />
+
+  <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <header class="navbar">
+    <a href="#" class="brand-title">My logo</a>
+    <i class="fas fa-bars"></i>
+  </header>
+
+  <container class="page">
 
     <section class="presentation">
-      <h2>I’m Alexandra. Glad to see you!</h2>
+      <h2>I’m Gonzalo. Glad to see you!</h2>
+
+      <article class="job">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
```

Comment:
Good job styling this portion of your portfolio 👏 , however it could be better if these texts  are aligned to stuck to the Figma design 💯 
![Capture d’écran 2023-03-29 à 2 44 39 AM](https://user-images.githubusercontent.com/80458199/228400713-8127af63-a1c5-4b15-8103-c9ba3ff059b3.png)